shared/Cargo.toml

```diff
@@ -31,22 +31,14 @@ bitflags = "1.3.2"
 ## non-wasm dependencies; these dependencies and the respective code is
 ## only activated through non-default features, which the Kernel enables, but
 ## not the actors.
-filecoin-proofs-api = { version = "14", default-features = false, optional = true }
 libsecp256k1 = { version = "0.7", optional = true }
-bls-signatures = { version = "0.13", default-features = false, optional = true }
+bls-signatures = { version = "0.12", default-features = false, optional = true }
 
 [dev-dependencies]
 rand = "0.8"
 rand_chacha = "0.3"
 serde_json = "1.0.56"
-multihash = { version = "0.16.3", default-features = false, features = ["multihash-impl", "sha2", "sha3", "ripemd"] }
+multihash = { version = "0.16.2", default-features = false }
```

Comment:
You can just remove this one and let it use the specification from the `dependencies` section.